rskj-core/src/test/java/co/rsk/peg/RegisterBtcTransactionIT.java

```diff
@@ -26,75 +26,125 @@
 import org.ethereum.config.blockchain.upgrades.ActivationConfig;
 import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
 import org.ethereum.core.*;
+import org.ethereum.crypto.ECKey;
 import org.ethereum.vm.PrecompiledContracts;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import java.util.*;
 
 class RegisterBtcTransactionIT {
-
     private final BridgeConstants bridgeConstants = BridgeMainNetConstants.getInstance();
     private final NetworkParameters btcParams = bridgeConstants.getBtcParams();
     private final BridgeSupportBuilder bridgeSupportBuilder = BridgeSupportBuilder.builder();
+    private Repository track;
+    private Repository repository;
+    private Block rskExecutionBlock;
+    private FederationSupport federationSupport;
+    private BridgeStorageProvider bridgeStorageProvider;
+    private BtcTransaction bitcoinTransaction;
+    private PartialMerkleTree pmtWithTransactions;
+    private int btcBlockWithPmtHeight;
+    private Transaction rskTx;
+    private RskAddress rskReceiver;
+    private BridgeSupport bridgeSupport;
+    private BridgeEventLoggerImpl bridgeEventLogger;
+    private Coin btcTransferred;
+
+    @BeforeEach
+    void setUp() throws Exception{
```

Comment:
there's a lot of static variables that can be declared at the class level instead of in the setup.
For example `rskTx`, `btcTransferred` (also I suggest to rename it to `minimumPeginTxValue`), etc

Reply:
done!